traveler-ios-ui/TravelerKitUI/ViewControllers/BookingItemDetailsViewController.swift

```diff
@@ -172,14 +172,19 @@ extension BookingItemDetailsViewController: WishlistAddDelegate {
     }
 
     public func wishlistAddDidFailWith(_ error: Error) {
-        let alert = UIAlertController(title: "Error", message: error.localizedDescription, preferredStyle: .alert)
-        let okAction = UIAlertAction(title: "Ok", style: .cancel, handler: { [unowned self] _ in
-            self.wishlistButton.isSelected = false
-        })
+        switch error {
+        case WishlistToggleError.unidentifiedTraveler:
```

Comment:
I get a `- {"errorCode":1001,"errorMessage":"Traveler not found"}` error that is not being covered or mapped correctly by this switch

Reply:
Any detail on how you got that error?